### PR TITLE
fix(dashboard): update dashboards, remove warnings

### LINKED ISSUE #6775: [Feature][Dashboard] Remove the warning when there's no data
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

As a user, when I see the initial status of any dashboard in Grafana, I can see the warning signs, which look like an error page.

I want to see more user-friendly dashboards when no data is collected.

image

### Description

It is mainly caused by the multi-value filter. When no data is collected, the SQL becomes

```
project in ()
```

which caused the syntax error.

image

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by d4x1**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [x] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summary
What does this PR do?
Update dashboards, remove warnings when there is no filter selected.

### Does this close any open issues?
Closes #6775 

### Screenshots
Include any relevant screenshots here.
##### Before
image

##### After
image

##### Before
image

##### After
image

##### Before
image

##### After
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by d4x1** [comment]
#### ContributorExperience
##### Before
image

##### After
image

#### Dora
##### Before
image

##### After
image

#### Dora By Team
##### Before
image

##### After
image

#### Engineering Overview
##### Before
image

##### After
image

**Comment by d4x1** [comment]
#### EngineeringThroughputAndCycleTime
##### Before
image

##### After
image

#### EngineeringThroughputAndCycleTimeTeamView
##### Before
image

##### After
image

#### GitHub
##### Before
image

##### After
image

#### GitLab
##### Before
image

##### After
image

#### Jenkins
##### Before
image

##### After
image

#### Jira
##### Before
image

##### After
image

#### Opsgenie
##### Before
image

##### After
image

**Comment by d4x1** [comment]
#### PagerDuty
##### Before
image

##### After
image


#### SonarQube
##### Before
image

##### After
image

#### TAPD
##### Before
image

##### After
image


#### Teambition
##### Before
image

##### After
image


#### Zentao
##### Before
image

##### After
image


#### WeeklyBugRetro
##### Before
image

##### After
image


#### WeeklyCommunityRetro
##### Before
image

##### After
image

#### GithubReleaseQualityAndContributionAnalysis
##### Before
image

##### After
image

File: grafana/dashboards/ComponentAndFileLevelMetrics.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
           "group": [],
           "metricColumn": "none",
           "rawQuery": true,
-          "rawSql": "SELECT file_path,\n    count(distinct author_name) AS cnt\nFROM commits\nJOIN commit_files\nJOIN repo_commits rc\n    ON commit_files.commit_sha = rc.commit_sha\n        AND commit_files.commit_sha = commits.sha\nWHERE repo_id IN ($repo_id)\n       AND $__timeFilter(commits.authored_date)\n       AND file_path REGEXP '($selected_path)'\nGROUP BY  file_path\nORDER BY  cnt DESC limit 10;",
+          "rawSql": "SELECT file_path,\n    count(distinct author_name) AS cnt\nFROM commits\nJOIN commit_files\nJOIN repo_commits rc\n    ON commit_files.commit_sha = rc.commit_sha\n        AND commit_files.commit_sha = commits.sha\nWHERE repo_id IN (${repo_id:sqlstring}+'')\n       AND $__timeFilter(commits.authored_date)\n       AND file_path REGEXP '(${selected_path:sqlstring}+'')'\nGROUP BY  file_path\nORDER BY  cnt DESC limit 10;",
```
Comment (Startrekzky) [inline grafana/dashboards/ComponentAndFileLevelMetrics.json:139]: I'm not sure if it works when a variable is used within a quote like 
```
AND file_path REGEXP '(${selected_path:sqlstring}+'')'
```

Comment (d4x1) [reply grafana/dashboards/ComponentAndFileLevelMetrics.json:139]: @

> I'm not sure if it works when a variable is used within a quote like
> 
> ```
> AND file_path REGEXP '(${selected_path:sqlstring}+'')'
> ```

Yes, it's a bug, regexp string should be handled in another way. I have updated.

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**